pkg/policy/l3_test.go

```diff
@@ -15,26 +15,56 @@
 package policy
 
 import (
+	"net"
+
+	"github.com/cilium/cilium/pkg/comparator"
 	"github.com/cilium/cilium/pkg/labels"
+	"github.com/cilium/cilium/pkg/node"
 
 	. "gopkg.in/check.v1"
 )
 
-func (ds *PolicyTestSuite) TestToBPFData(c *C) {
-	cidrPolicy := NewCIDRPolicy()
-	m := cidrPolicy.Ingress
+// init prepares the node's allocation range so that the ``ToBPFData()`` calls
+// below can successfully fetch the cluster range.
+func init() {
```

Comment:
I would prefer that you use the TearDown function to make sure the `node.Set..` doesn't affect other tests

Reply:
I had to introduce a new `Uninitialize()` function in the node package for the teardown. Done this locally, will push out soon.